src/main/java/gov/cabinetoffice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -248,7 +251,7 @@ private Cookie addCustomJwtCookie(final HttpServletResponse response,
                                       final boolean isAdmin) {
         final Map<String, String> customJwtClaims = oneLoginService.generateCustomJwtClaims(userInfo, idToken);
         final String customServiceJwt = customJwtService.generateToken(customJwtClaims, isAdmin);
-        final Cookie customJwt = WebUtil.buildSecureCookie(userServiceCookieName, customServiceJwt);
+        final Cookie customJwt = WebUtil.buildSecureCookie(userServiceCookieName, userServiceCookieDomain, customServiceJwt);
```

Comment:
as per comment above, we should swap these round
```suggestion
        final Cookie customJwt = WebUtil.buildSecureCookie(userServiceCookieName, customServiceJwt, userServiceCookieDomain);
```

Reply:
Will change this in next push!